cast/src/receiver/layout/hc-lovelace.ts

```diff
@@ -45,22 +44,14 @@ class HcLovelace extends LitElement {
       deleteConfig: async () => undefined,
       setEditMode: () => undefined,
     };
-    return this.lovelaceConfig.views[index].panel
```

Comment:
Why is this check no longer necessary?

Reply:
We are doing this check in HUI-View now.